numpy/_core/tests/test_numeric.py

```diff
@@ -3210,7 +3217,8 @@ def test_tol_warnings(self):
 
         for i in b:
             for j in b:
-                # Making sure that i and j are not both numbers, because that won't create a warning
+                # Making sure that i and j are not both numbers,
+                #because that won't create a warning
```

Comment:
```suggestion
                # because that won't create a warning
```
From [PEP-8](https://peps.python.org/pep-0008/) on comments:

> They should start with a # and a single space